app/models/user/profile.rb

```diff
@@ -5,18 +5,25 @@ class User::Profile < ApplicationRecord
 
   delegate :to_param, to: :user
 
+  COUNT_TO_SHOW_TAB = 4
+
+  def self.count_to_show_tab
+    COUNT_TO_SHOW_TAB
+  end
+
   memoize
   def solutions_tab?
-    user.solutions.published.count > 3
+    user.solutions.published.count >= self.class.count_to_show_tab
   end
 
   memoize
   def testimonials_tab?
-    user.mentor_testimonials.published.count > 3
+    user.mentor_testimonials.published.count >= self.class.count_to_show_tab
   end
 
   memoize
   def contributions_tab?
-    user.reputation_tokens.where(category: %i[building authoring maintaining]).exists?
+    user.reputation_tokens.where(category: %i[building authoring
```

Comment:
@iHiD I needed to modify this to make the tests pass. Is this correct? Or should I change the test to have a corresponding reputation token awarded to the exercise authorship?

Reply:
Change the test pls :)

Reply:
We then need a way to ensure that a reputation token is awarded for exercise authorships which we could add in later. The authorship tab values don't have anything to do with awarded reputation tokens, but rather they are accessing the exercise authorships and contributions model directly.

Reply:
> We then need a way to ensure that a reputation token is awarded for exercise authorships

Yeah - it is. It's all automated through the GH syncing process. You get two reputation tokens for authoring - one for the authoring itself and one for the PR that creates it.